docs/4-mandat.adoc

```diff
@@ -13,6 +13,6 @@ Sørg for at sentrale fagressurser blir involvert i beskrivelsen av punktene i f
 
 Ta kontakt med andre etater for å bli kjent med andre sine erfaringer. Etatene som har vært med i SKATE- arbeidet har for eksempel vært villig til å dele informasjon om prosesser, organisering av arbeid, rammeverk og  bruk av verktøy.
 
-Delta på kurs og arrangementer som gir mer informasjon om temaet, og som gir muligheter for å delta på faglige nettverk. Eksempler på kurs finnes på https://www.difi.no/opplaeringstilbud/kommende-arrangementer[Difi sin arrangementsoversikt]. Difi arrangerer også møter i https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/nasjonal-arkitektur/informasjonsforvaltning/fagleg-arena-informasjonsforvaltning[faglig arena for informasjonsforvaltning] som det kan være nyttig å delta på for å få innspill til eget arbeid i orden i eget hus.
+Delta på kurs og arrangementer som gir mer informasjon om temaet, og som gir muligheter for å delta på faglige nettverk. Eksempler på kurs finnes på https://https://www.digdir.no/arrangement/arrangementsoversikt/692?field_end_date_value=1[Digitaliseringsdirektoratet sin arrangementsoversikt]. Digitaliseringsdirektoratet arrangerer også møter i https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/nasjonal-arkitektur/informasjonsforvaltning/fagleg-arena-informasjonsforvaltning[faglig arena for informasjonsforvaltning] som det kan være nyttig å delta på for å få innspill til eget arbeid i orden i eget hus.
 
-Ta gjerne kontakt med Difi eller Brønnøysundregistrene footnote:[Brreg: https://www.brreg.no/hvordan-kan-vi-hjelpe-deg/ Difi: informasjonsforvaltning@difi.no] hvis du ønsker noen utenfra til å fortelle om orden i eget hus og datadeling som et nasjonalt løft.
+Ta gjerne kontakt med Digitaliseringsdirektoratet eller Brønnøysundregistrene footnote:[Digitaliseringsdirektoratet: informasjonsforvaltning@digdir.no] hvis du ønsker noen utenfra til å fortelle om orden i eget hus og datadeling som et nasjonalt løft.
```

Comment:
jeg foreslår at vi sjekker om Brønnøysundregistrene (BR) fortsatt ønsker å være nevnt her, samt at vi vurderer om det i så fall kun er dem som skal nevnes eller om det også er andre virksomheter som bør nevnes blant dem som kan kontaktes for å få innspill til Orden i eget hus-arbeid. BR har tradisjon for samarbeid og deling av erfaringer, og jeg regner med at de ønsker å fortsette med det, men teksten her tok kanskje utgangspunkt i "arbeidsdelingen" slik den var før opprettelsen av Digdir? Vi ser jo at det er flere enn BR som har begynt å få erfaringer som kan deles, så vi bør vurdere om vi kan legge til flere på listen over aktuelle kandidater som kan kontaktes